lookout/style/format/visualizer/README.md

```diff
@@ -10,6 +10,10 @@ in the visualizer directory to install dependencies.
 
 Train a model and put it in the `models` directory. Name it `model.asdf`.
 
+It could require to install
```

Comment:
Listed in [requirements.txt](https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/visualizer/requirements.txt)